docs/aws/README.md

```diff
@@ -1,46 +1,108 @@
 
 # AWS
-## Installation
-Karpenter's pod requires AWS Credentials to read or modify AWS resources in your account. We recommend using IRSA (IAM Roles for Service Accounts) to manage these permissions.
-
+This guide will provide a complete Karpenter installation for AWS. These steps are opinionated and may need to be adapted for your use case.
+## Environment
 ```bash
+CLOUD_PROVIDER=aws
 AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
-CLUSTER_NAME=<your_cluster>
-REGION=<your_region>
+CLUSTER_NAME=$USER-karpenter-demo
+AWS_DEFAULT_REGION=us-west-2
 ```
 
-### Create Karpenter IAM Resources
-This command will create IAM resources used by Karpenter. For production use, please review and restrict these permissions as necessary.
+### Create a Cluster
+Skip this step if you already have a cluster.
+```bash
+eksctl create cluster \
+--name ${CLUSTER_NAME} \
+--version 1.18 \
+--region ${AWS_DEFAULT_REGION} \
+--nodegroup-name karpenter-aws-demo \
+--node-type m5.large \
+--nodes 1 \
+--nodes-min 1 \
+--nodes-max 10 \
+--managed
+```
+
+### Create IAM Resources
+This command will create IAM resources used by Karpenter. We recommend using Cloud Formation and IAM Roles for Service Accounts to manage these permissions. For production use, please review and restrict these permissions for your use case.
 ```bash
-// TODO, point to github raw uri
 aws cloudformation deploy \
   --stack-name Karpenter \
   --template-file ./docs/aws/karpenter.cloudformation.yaml \
```

Comment:
Do we still need the raw URI link?

Reply:
I tried to make this work, IIUC, aws cloudformation deploy only works with local files.